src/redux/actions/concepts/addBulkConcepts/index.js

```diff
@@ -50,7 +50,7 @@ export const fetchFilteredConcepts = (source = 'CIEL', query = '') => async (
     dispatch(isFetching(false));
   } catch (error) {
     dispatch(isFetching(false));
-    notify.show('an error occurred, reload your browser', 'error', 3000);
+    notify.show('An error occurred, please try reloading the page', 'error', 3000);
```

Comment:
I think you should add something like `An error occurred with your internet connection, please fix it and try reloading the page`. What do you think?

Reply:
@Duncanian that's a good suggestion. Let me change it to that.